util/network-devp2p/src/node_table.rs

```diff
@@ -111,6 +111,14 @@ impl NodeEndpoint {
 		self.to_rlp(rlp);
 	}
 
+	pub fn to_enr(&self, key: &parity_crypto::publickey::Secret) -> enr::Enr {
```

Comment:
It seems that no functions here have docs, but I'd rather see new ones to have some docs than to continue this bad tradition.